From cf69e14a77b38966ef477c26ddeddc8680db7066 Mon Sep 17 00:00:00 2001
From: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Date: Mon, 1 Feb 2021 14:52:29 +0200
Subject: [PATCH 53/78] s32g274a-evb: qspi: Update clock speed according to SoC
 Rev 2

The default QSPI maximum frequency is now 200 MHZ on S32G Rev 2.
The QSPI frequency will be either way modified by u-boot, depending
on SoC Rev, therefore, this change has no functional impact.

Issue: ALB-6419
Signed-off-by: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g274a-evb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g274a-evb.dts b/arch/arm64/boot/dts/freescale/fsl-s32g274a-evb.dts
index d97018daecb3..42b9b5a181af 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g274a-evb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g274a-evb.dts
@@ -59,7 +59,7 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 		compatible = "jedec,spi-nor";
-		spi-max-frequency = <133000000>;
+		spi-max-frequency = <200000000>;
 		spi-tx-bus-width = <8>;
 		spi-rx-bus-width = <8>;
 		reg = <0>;
-- 
2.25.1

